### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -7561,13 +7561,18 @@ fertily->fertility
 fetaure->feature
 fetaures->features
 feture->feature
+fewg->fugue
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest few

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -7561,13 +7561,18 @@ fertily->fertility
...
+fewsha->fuchsia
+fezent->pheasant
 fhurter->further
 fials->fails
 fianite->finite
 fianlly->finally
+ficks->fix
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest flicks

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -7597,12 +7602,16 @@ filetests->file tests
 fileystem->filesystem
 fileystems->filesystems
 filiament->filament
+filing->filling
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: This is still a real word, https://dictionary.cambridge.org/dictionary/english/filing

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -7597,12 +7602,16 @@ filetests->file tests
...
+fillay->fillet
 fille->file, fill, filled,
+fillement->filament
 filles->files, fills, filled,
 fillowing->following
 fillung->filling
 filnal->final
 filname->filename
+filo->phyllo
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Remove, style, also the same word: https://en.wikipedia.org/wiki/Filo

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: delete

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Deleted

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Thanks, I suspect few is more likely as a word.

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Comment by Gelma** [comment]
> Just swapping the fewg suggestions order please.

Done

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -7756,14 +7756,19 @@ fertily->fertility
 fetaure->feature
 fetaures->features
 feture->feature
-fewd->feud, few,
+fewd->few, feud
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Trailing comma as per Travis.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -7756,14 +7756,19 @@ fertily->fertility
...
+fewg->few, fugue
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Trailing comma as per Travis.

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**